submissions/Zakkarat/memory-pair-game/app.js

```diff
@@ -0,0 +1,74 @@
+const cards = document.getElementsByClassName('flipper');
+const container = document.getElementsByClassName('container')[0];
+const modalWindow = document.getElementsByClassName('modal')[0];
+const modalContent = document.getElementsByClassName('modal-content')[0];
+const button = document.getElementsByTagName('button')[0];
+let founded = [];
+let images = ['champ1', 'champ2', 'champ3', 'champ4', 'champ5'];
+let check = [];
+let checkFirst = true;
+let checkSecond = true;
+let prevTarget = "/img/back.jpg";
+prepareArray();
+container.addEventListener('click', function(elem) {if(checkFirst && checkSecond && elem.target.src != prevTarget) cardFlip(elem)});
```

Comment:
Do you use prettier? You if statement here looks weird.

Reply:
Thank you! Now everything is prettified.